codon/parser/common.cpp

```diff
@@ -367,6 +369,28 @@ std::shared_ptr<ImportFile> getImportFile(const IFilesystem *fs,
         paths.emplace_back(fs->canonical(path));
     }
   }
+  if (paths.empty()) {
+    // Load a plugin maybe
+    auto path = parentRelativeTo / what;
+    if (fs->exists(path / "plugin.toml") &&
+        fs->exists(path / "stdlib" / "__init__.codon")) {
+      bool failed = false;
+      if (cache->compiler && !cache->compiler->isPluginLoaded(path)) {
+        LOG("Loading plugin {}", path);
```

Comment:
Don't think we need to log this, or we can have it controlled by a flag like e.g. `-plugin-verbose`.